asmcomp/amd64/emit.mlp

```diff
@@ -615,14 +615,16 @@ let emit_instr fallthrough i =
         end;
         cfi_remember_state ();
```

Comment:
Can remove this,
```suggestion
```

Reply:
We need it to restore the offsets to the last know good value after the call. It appears that it works fine in this specific example, but that might just be a coincidence. I would keep it around since any future change might make the code break.

Reply:
Agree.